src/common/storage/src/operator.rs

```diff
@@ -255,6 +255,7 @@ fn init_obs_operator(cfg: &StorageObsConfig) -> Result<Operator> {
 #[derive(Clone, Debug)]
 pub struct StorageOperator {
     operator: Operator,
+    share_endpoint_address: Option<String>,
```

Comment:
We should not touch this struct.

Reply:
we need some way to get the share endpoint address when creating FuseTable, where will be more suitable?

Reply:
How about placing it in table options?

Reply:
`TableInfo` is return by meta server, and meta is unaware of share endpoint.

Reply:
I think we need to refactor the `get_table_info` API and insert `endpoint_address` there.